secondhw/src/main/java/io/codelex/loops/practice/FizzBuzz.java

```diff
@@ -0,0 +1,33 @@
+package io.codelex.loops.practice;
+
+import java.util.Scanner;
+
+public class FizzBuzz {
+    public static void main(String[] args) {
+
+        Scanner input = new Scanner(System.in);
+        System.out.print("Please, enter positive number: ");
+        int number = input.nextInt();
+
+        System.out.println("Max value? " + number);
+        for (int i = 1; i <= number; i++) {
+            if (i % 15 == 0) {
+                System.out.print("FizzBuzz ");
+                if (i % 20 == 0) {
```

Comment:
lai šis neatkārtojas ķšiet jau var iznest uz vienu reizi uz loop beigām? :thinking:

Reply:
hmm, laikam gan, bet pirms tam kaut kā galīgi nesanāca, laikam ar iekavām kaut kas nabija pareizi.
check!